### DISCUSSION TITLE
Constant Error Display When Initiating a Transaction and Questions Regarding Custom Naming of ABI Methods

### DISCUSSION THREAD

**Discussion by csschan** [rainbow-me/rainbowkit#2103] [Q&A]
![WX20240716-101953@2x](https://github.com/user-attachments/assets/2e824bd6-154c-41b4-95d7-a62aa7899cb8)
When initiating a transaction, I constantly encounter an error as shown in the attached image. Additionally, I have another question: Is it possible to customize the naming of ABI methods?

测试代码如下图
![WX20240716-102254@2x](https://github.com/user-attachments/assets/c19f3f47-25c6-49af-a549-d65a42121c86)

**Comment by DanielSinclair** [comment]
These Wagmi calls are based off of Abitype. I'd recommend you reference some of the docs [here](https://abitype.dev/) and open an issue on the Wagmi project for the type issues you are seeing. It seems like the `mint` function for this contract does not accept `args`. You can reference our NFT minting example here: https://codesandbox.io/p/sandbox/github/rainbow-me/rainbowkit/tree/main/examples/with-next-mint-nft
